Automattic/jetpack
# Subscriber Login: Add styling support to the Subscriber Login block

Part of https://github.com/Automattic/jetpack/issues/34884

## Proposed changes:

It:
- adds styling support to the Subscriber Login block (aligning, spacing, typography, and colors)
- adds some keywords to make the block easier to search
- make the custom class work in the editor
- removes not necessary `editor.scss` file
- replaces `@since 0.0.1` with `@since $$next-version$$` to allow the Jetpack release process to overwrite it with the correct version
- checks if the class exists before using it

### Other information:


## Jetpack product discussion

N/A

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

Try to add some styling settings to the Subscriber Login block making sure it works as expected.
